client/telemetry/src/lib.rs

```diff
@@ -18,86 +18,50 @@
 
 //! Telemetry utilities.
 //!
-//! Calling `init_telemetry` registers a global `slog` logger using `slog_scope::set_global_logger`.
-//! After that, calling `slog_scope::with_logger` will return a logger that sends information to
-//! the telemetry endpoints. The `telemetry!` macro is a short-cut for calling
-//! `slog_scope::with_logger` followed with `slog_log!`.
+//! `Telemetry` objects can be created through its constructor `Telemetry::new()`, or through a
```

Comment:
Please describe how it works from a high level and not how to call `new`.

Reply:
I have added a bit more details... This sentence is not really explaining how to call `new`, it's comparing the `Telemetry::new` constructor from the `Telemetries.build_telemetry`.

@wheresaddie can you help here? :grin:

Reply:
I re-worked it: https://github.com/paritytech/substrate/pull/7463/files#diff-5d81cc2d5a422bf08fab539e181adf457a86ca2a1680602840668c8713846724

It's more high level. Let me know if it is better